keyboards/gray_studio/cod67/rules.mk

```diff
@@ -17,15 +17,15 @@ FIRMWARE_FORMAT = bin
 # Build Options
 #   change yes to no to disable
 #
-BOOTMAGIC_ENABLE = yes      # Virtual DIP switch configuration(+1000)
-MOUSEKEY_ENABLE = no        # Mouse keys(+4700)
+BOOTMAGIC_ENABLE = no       # Virtual DIP switch configuration(+1000)
```

Comment:
if you have flash space  `lite` is a good option incase people forget to put a RESET code on their keymaps and can't access the reset switch.

Reply:
The default bootloader has hold Esc to trigger DFU, so I left it off. If enabling lite bootmagic won't interfere with that then I can put it on a different key maybe?